gk/main.c

```diff
@@ -2720,7 +2719,7 @@ gk_log_flow_state(const char *src_addr,
 		flow.f.v6.dst = dst.ip.v6;
 	}
 
-	mb = get_responsible_gk_mailbox(&flow, gk_conf);
+	mb = get_responsible_gk_mailbox(rss_ip_flow_hf(&flow, 0, 0), gk_conf);
```

Comment:
Have a second patch to also pass this computed hash value to the GK block.

Reply:
As the computed hash value isn't used by the GK blocks (i.e., in ``log_flow_state()``), do we also need to change the function call ``rte_hash_lookup()`` to ``rte_hash_lookup_with_hash()``?

Reply:
Yes. But we don't need to batch it.